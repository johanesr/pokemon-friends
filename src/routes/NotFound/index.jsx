import './styles.scss';

function NotFound() {
  return (
    <div className="not-found-wrapper">
      Not Found
    </div>
  )
}

export default NotFound;