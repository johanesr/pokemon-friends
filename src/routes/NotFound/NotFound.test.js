import { shallow } from 'enzyme';
import NotFound from './index';

const wrapper = shallow(<NotFound />);

it("renders NotFound Container correctly", () => {
  expect(wrapper).toMatchSnapshot(); // check if a container renders correctly
});