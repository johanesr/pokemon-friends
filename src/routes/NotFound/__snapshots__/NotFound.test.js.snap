// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NotFound Container correctly 1`] = `
<div
  className="not-found-wrapper"
>
  <div>
    Page is not found
  </div>
  <div>
    <Link
      to="/list"
    >
      Return to home page?
    </Link>
  </div>
</div>
`;
